propagators/opentelemetry-propagator-opentracing/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@opentelemetry/propagator-opentracing",
+  "version": "0.12.0",
```

```suggestion
  "version": "0.12.1",
```
But most likely it will move further to 0.13.0

---

I preemptively went to 0.13.0.